src/gui/tray/Window.qml

```diff
@@ -221,6 +239,13 @@ Window {
                                     }
                                 }
                                 onClicked: UserModel.addAccount()
+
+                                Accessible.role: Accessible.MenuItem
+                                Accessible.name: qsTr("Add new account")
+                                Accessible.description: qsTr("Opens setup window to add a new account connection")

```

Comment:
@tyrylu @Adriorjalesvidal would you say the "Add new account" action needs an additional description at all?